browser/main/lib/dataApi/attachmentManagement.js

```diff
@@ -286,13 +287,24 @@ function handleAttachmentDrop (codeEditor, storageKey, noteKey, dropEvent) {
             isImage: true
           }))
         } else {
-          return fixRotate(file)
-            .then(data => copyAttachment({type: 'base64', data: data, sourceFilePath: file.path}, storageKey, noteKey)
-              .then(fileName => ({
+          return getOrientation(file)
+            .then((orientation) => {
+              if (orientation === -1) { // The image rotation is correct and does not need adjustment
+                return copyAttachment(file.path, storageKey, noteKey)
```

Comment:
Can you change it to `filePath` too? Thanks!

Reply:
Change made as requested, alongside the same change in other parts of the function. Retested with same images.